WordPress/Classes/ViewRelated/Me/Account Settings/AccountSettingsViewController.swift

```diff
@@ -356,8 +356,12 @@ private class AccountSettingsController: SettingsController {
     }
 
     private var localizedErrorMessageForAtomicSites: String {
-        NSLocalizedString("To close this account now, contact our support team.",
-                                 comment: "Error message displayed when unable to close user account due to having active atomic site.")
+        // Based on https://github.com/Automattic/wp-calypso/pull/65780
+        NSLocalizedString(
+            "accountSettings.closeAccount.error.atomicSite",
+            value: "This user account cannot be closed because it has active purchases. Please contact our support team.",
```

Comment:
The current copy might be sufficient. 

Although, we could consider using even more precise language that [Apple is using](loper.apple.com/support/offering-account-deletion-in-your-app), to remove any ambiguity about the possibility of closing the account.

```suggestion
            value: "This user account cannot be closed immediately because it has active purchases. Please contact our support team to finish deleting the account.",
```

Reply:
Neat! Applied. Thanks.